pr-data.csv

```diff
@@ -1941,7 +1941,7 @@ https://github.com/apache/hadoop,cc2babc1f75c93bf89a8f10da525f944c15d02ea,hadoop
 https://github.com/apache/hadoop,a585a73c3e02ac62350c136643a5e7f6095a3dbb,hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-app,org.apache.hadoop.mapreduce.jobhistory.TestJobHistoryEventHandler.testUnflushedTimer,OD,,,
 https://github.com/apache/hadoop,a585a73c3e02ac62350c136643a5e7f6095a3dbb,hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core,org.apache.hadoop.mapreduce.lib.db.TestSplitters.testBooleanSplitter,OD,,,
 https://github.com/apache/hadoop,cc2babc1f75c93bf89a8f10da525f944c15d02ea,hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-jobclient,org.apache.hadoop.mapreduce.lib.input.TestCombineFileInputFormat.testSplitPlacement,ID,,,
-https://github.com/apache/hadoop,cc2babc1f75c93bf89a8f10da525f944c15d02ea,hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-jobclient,org.apache.hadoop.mapreduce.lib.input.TestCombineFileInputFormat.testSplitPlacementForCompressedFiles,ID,,,
+https://github.com/apache/hadoop,cc2babc1f75c93bf89a8f10da525f944c15d02ea,hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-jobclient,org.apache.hadoop.mapreduce.lib.input.TestCombineFileInputFormat.testSplitPlacementForCompressedFiles,ID,DeveloperFixed,,could not reproduce: https://github.com/apache/hadoop/commit/551615fa13f65ae996bae9c1bacff189539b6557
```

Comment:
If you want to claim `DeveloperFixed`, you'd need to find the exact commit SHA where the test was fixed, i.e., it can fail in the previous commit `SHA^` but always passes in `SHA`.